cid/helpers/quicksight/__init__.py

```diff
@@ -81,6 +81,9 @@ def user(self) -> dict:
             if not self._user:
                 # If no user match, ask
                 self._user = self.select_user()
+            if not self._user:
```

Comment:
We have to detect the username from the session

Reply:
yes. indeed